app/Eloquent/PermissionRole.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Eloquent;
+
+use Illuminate\Database\Eloquent\Model;
+
+class PermissionRole extends Model
+{
+    protected $table = 'permission_role';
+
+    protected $fillable = [
+    	'permission_id',
+    	'role_id',
+    ];
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.DisallowTabIndent.TabsUsed</br>**Severity**: error</br>**File**: app/Eloquent/PermissionRole.php</br>Spaces must be used to indent lines; tabs are not allowed